plugins/modules/ecs_taskdefinition.py

```diff
@@ -61,7 +476,7 @@
         type: str
     task_role_arn:
         description:
-            - The Amazon Resource Name (ARN) of the IAM role that containers in this task can assume. All containers in this task are granted
+            - The Amazon Resource Name (ARN) of the IAM role that containers in this task can assume.All containers in this task are granted
```

Comment:
```suggestion
            - The Amazon Resource Name (ARN) of the IAM role that containers in this task can assume.  All containers in this task are granted
```

Reply:
Done, thank you very much for reviewing!